pandas/core/apply.py

```diff
@@ -18,6 +18,8 @@
 if TYPE_CHECKING:
     from pandas import DataFrame, Series, Index
 
+ResType = Dict[int, Any]
```

Comment:
ResultType maybe

Reply:
That was what I tried first, but lines wrap prettier with the shorter name